### PR TITLE
Add acceptance test for typescript app

### PR DESCRIPTION
**Pull request by jdolitsky**
Add a simple test app to see that the built version of the package can be imported in another app successfully.

Removed `build` step in CI (`test` runs `build`).

Also, added timezone for Jest tests (which fail if run in timezone other than EST).

### CODE REVIEW

**Comment by jdolitsky** [comment]
Also, just added GitHub Actions workflow for incoming PRs

File: package.json [hunk 1]
```diff
@@ -13,14 +13,15 @@
   "scripts": {
     "codegen": "scripts/codegen.sh",
     "build": "scripts/build.sh",
+    "acceptance": "scripts/acceptance.sh",
     "publish": "scripts/publish.sh",
     "fix": "run-s fix:*",
     "fix:prettier": "prettier \"src/**/*.ts\" --write",
     "fix:lint": "eslint src --ext .ts --fix",
-    "jest": "jest",
+    "jest": "TZ=America/New_York jest",
```
Comment (zeppelinnn) [inline package.json:21]: why is this needed?

Comment (jdolitsky) [reply package.json:21]: The test mock responses were built for the EST timezone, so running with a different timezone fails.

See this test failure: https://github.com/getoctane/octane-node/runs/3616483350?check_suite_focus=true

cc @salemhilal

**Comment by zeppelinnn** [comment]
@jdolitsky Could you try reverting back to the previous version (where the failure happened) and verify that
1) Build fails without fix
2) Running the acceptance test catches this failure which is blocking

In theory you can just undo the changes made in https://github.com/getoctane/octane-node/pull/12 if that's more straightforward to do.

**Comment by jdolitsky** [comment]
@zeppelinnn - please see 4d233f4 and associated test failure: https://github.com/getoctane/octane-node/actions/runs/1246311618

Fails as expected with
```
node_modules/octane-node/codegen/api.d.ts(12,23): error TS2688: Cannot find type definition file for 'codegen/custom'.
```

**Review by zeppelinnn — APPROVED** [review]

File: testdata/simple-typescript-app/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import util from 'util';
+import Octane from 'octane-node';
+
+const octane = new Octane('abc123');
+console.log(util.inspect(octane, {depth: 0}));
```
[the remark this answers is not in the document]
Comment (jdolitsky) [reply testdata/simple-typescript-app/src/index.ts:5]: I think so, it shows that the structure of the Octane object is as expected
```
Octane {
  customers: [Customers],
  measurements: [Measurements],
  meters: [Meters],
  pricePlans: [PricePlans]
}
```
